src/core/graphql/graphql.options.ts

```diff
@@ -96,14 +92,10 @@ export class GraphqlOptions implements GqlOptionsFactory {
     };
   }
 
-  context = ({
-    req: request,
-    reply: response,
-  }: ServerContext): Partial<GqlContextType> => {
+  context = ({ req: request }: ServerContext): Partial<GqlContextType> => {
     return {
       [isGqlContext.KEY]: true,
       request,
-      response,
```

Comment:
This function is just the "user" context. Which is merged with the `ServerContext`. Since it already had `response`, we don't need to override it with itself here. This could've been needed with the old driver.

Just to note as well, the yoga server declares `request` as the web standard `Request` type.
We'd rather use the _Fastify_ `Request` instead instead, at least for now to keep compatibility.
So we have the "server context" declare a temp `req` so yoga's `request` doesn't conflict. Then the "user context" comes after both of these and can override yoga's built in `request` property.